maas/resource_maas_vm_host.go

```diff
@@ -63,6 +63,13 @@ func resourceMAASVMHost() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
+			"certificate": {
+				Type:          schema.TypeString,
+				Optional:      true,
```

Comment:
Should we consider adding `Computed: true` as wel to cover the case when we only provide the trust password and MAAS is creating a cert/key combo for the VM host?

Reply:
of course, in that case we need to also update the read function accordingly.